mi/dataset/parser/common_regexes.py

```diff
@@ -35,7 +35,7 @@
 ASCII_HEX_CHAR_REGEX = r'[0-9A-Fa-f]'
 
 # A regex used to match a date in the format YYYY/MM/DD
-DATE_YYYY_MM_DD_REGEX = r'(\d{4})/(\d{2})/(\d{2})'
+DATE_YYYY_MM_DD_REGEX = r'(\d{4})\/(\d{2})\/(\d{2})'
```

Comment:
how did this ever work before?

Reply:
That's a good question.  I believe it's the difference between the python re package and the regex testers I was using online.  I believe it worked because re took the '/' as literals as it should have, but the tools saw that the '/' was an error in the regex that it was used.  The tool said it needed to be escaped.  Since the '/' are supposed to be taken literally, I saw no harm in adding the '\' escape character.  This was not needed to fix this issue, but it was obvious that it was safer to escape those '/''s.